### PR TITLE
docs: update Apache Kafka Scaler Doc for multi topic lag

### PR DESCRIPTION
**Pull request by PaulLiang1**
Update kafka scaler doc for change  https://github.com/kedacore/keda/pull/2409

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

### CODE REVIEW

**Review by tomkerkhove — APPROVED** [review]
LGTM, but waiting to merge until feature is merged.

**Review by JorTurFer — APPROVED** [review]

**Comment by PaulLiang1** [comment]
> But it did not highlight the difference between auto discovery vs multi-tigger;

@PaulLiang1 do you think you can a few words about this to the docs? So users are aware of the consequences?

_Originally posted by @zroubalik in https://github.com/kedacore/keda/issues/2409#issuecomment-1004013964_

---- 
Added extra info added as requested from above comment

File: content/docs/2.6/scalers/apache-kafka.md [hunk 1]
```diff
@@ -33,7 +36,27 @@ triggers:
 
 - `bootstrapServers` - Comma separated list of Kafka brokers "hostname:port" to connect to for bootstrap.
 - `consumerGroup` - Name of the consumer group used for checking the offset on the topic and processing the related lag.
-- `topic` - Name of the topic on which processing the offset lag.
+- `topic` - Name of the topic on which processing the offset lag. (Optional)
+
+  > **Note:**
```
Comment (zroubalik) [inline content/docs/2.6/scalers/apache-kafka.md:41]: Could you please move this Note down below the list of parameters? The note itself is great.

File: content/docs/2.6/scalers/apache-kafka.md [hunk 2]
```diff
@@ -33,7 +36,27 @@ triggers:
...
+  > - When there are **active** consumer instances, _all topics_ includes:
+  >   - Topics the consumer is *currently* subscribing to;
+  >   - Topics that the consumer group *had prior commit history* (up to retention period for `__consumer_offset`, default to 7 days, see [KIP-186](https://cwiki.apache.org/confluence/display/KAFKA/KIP-186%3A+Increase+offsets+retention+default+to+7+days));
+  > - When there are **no active** consumer instances, _all topics_ only includes topics that the consumer group *had prior commit history*;
+  > ---
+  > An edge case exists where scaling could be **effectively disabled**:
+  >    - Consumer never makes a commit (no record in `__consumer_offset`);
+  >    - and `ScaledObject` had `minReplicaCount` as 0;
+  >
+  >   In such case, Keda could scale the consumer down to 0 when there is no lag and won't be able scale up due to the topic could not be auto discovered.
```
Comment (zroubalik) [inline content/docs/2.6/scalers/apache-kafka.md:53]:
```suggestion
  >   In such case, KEDA could scale the consumer down to 0 when there is no lag and won't be able scale up due to the topic could not be auto discovered.
```
Nit: please use KEDA not Keda :)

**Review by zroubalik — COMMENTED** [review]
Looking good, just a minor nit.

File: content/docs/2.6/scalers/apache-kafka.md (hunk 1 above)
Comment (PaulLiang1) [reply content/docs/2.6/scalers/apache-kafka.md]: moved.

File: content/docs/2.6/scalers/apache-kafka.md (hunk 2 above)
Comment (PaulLiang1) [reply content/docs/2.6/scalers/apache-kafka.md]: renamed.

**Review by zroubalik — APPROVED** [review]
LGTM

Thanks a lot @PaulLiang1!

**Pull request merged**